### PR TITLE
Remove build from postinstall script

### PR DESCRIPTION
**Pull request by restlessbit**
`underdog-styles` fails on install because it automatically runs a build in the `postinstall` script, but npm does not install dev dependencies on install which are required in order to build static assets.

Compiling assets is required before starting when running the Express server, so the build step has been moved to the Heroku startup script in `Procfile`.

/cc @underdogio/engineering

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: npm start
+web: npm run build && npm start
```
Comment (brettlangdon) [inline Procfile]: What this means is that whenever a new Dyno is spun up for `underdog-styles` that we will be running `npm run build`.

For example, if the dyno goes to sleep, and then we access the site, we will have to wait for `npm run build` to complete before we'll be able to access it.

**Comment by brettlangdon** [comment]
https://devcenter.heroku.com/articles/nodejs-support#heroku-specific-build-steps

**Comment by restlessbit** [comment]
Never thought I'd say this, but thanks Heroku.

**Comment by restlessbit** [comment]
Updated.

**Comment by brettlangdon** [comment]
haha, yeah

lgtm

**Pull request merged**